### PR TITLE
Switch SearchBar and ChromeScopes Proptypes to flow props

### PR DESCRIPTION
**Pull request by inyigo**
hi! first PR, hoping to help :) 

Associated Issue: [#2804](https://github.com/devtools-html/debugger.html/issues/2804)

## Summary of Changes

 Changed PropTypes to Flow Props

## Test Plan

- [x] yarn run test-all
- [x] yarn run flow

### CODE REVIEW

File: src/components/SecondaryPanes/ChromeScopes.js [hunk 1]
```diff
@@ -32,6 +31,12 @@ function createNode(name, path, contents) {
 }
 
 class Scopes extends Component {
+  props: {
+    scopes: Array<Object>,
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/ChromeScopes.js]: I think you can import `Scope` from flow-types/debugger.html

File: src/components/SecondaryPanes/ChromeScopes.js [hunk 2]
```diff
@@ -32,6 +31,12 @@ function createNode(name, path, contents) {
...
+    loadedObjects: Map<string, any>,
+    loadObjectProperties: Function,
+    pauseInfo: Object
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/ChromeScopes.js]: perhaps the same w/ PauseInfo

File: src/components/SecondaryPanes/ChromeScopes.js [hunk 3]
```diff
@@ -32,6 +31,12 @@ function createNode(name, path, contents) {
 }
 
 class Scopes extends Component {
+  props: {
+    scopes: Array<Object>,
+    loadedObjects: Map<string, any>,
+    loadObjectProperties: Function,
```
Comment (jasonLaster) [inline src/components/SecondaryPanes/ChromeScopes.js]: lets type the params: `(string) => void` or something like that

**Comment by inyigo** [comment]
@jasonLaster thanks, I've addressed the comments. 

Would you happen to know if there's any difference between `Array<Scope>` and `Scope[]` ?

**Comment by jasonLaster** [comment]
> Would you happen to know if there's any difference between Array<Scope> and Scope[] ?



nope, none

**Pull request merged**